src/components/Paginate.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { useEffect, useState } from "react";
+
+const findNumberOfPages = (
```

Comment:
Slightly better name: `getPageCount`, which aligns more closely to `resultsCount`, the variable in the previous PR comments. Methods/fn's prefixed with `find` imply `.find()` is being used. `get` is more generic and doesn't have this implication.

Reply:
That makes sense. e89a619294c798f2c7224784f816755f31725541 addresses this.